frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -142,7 +143,14 @@ class TableInteractive extends Component {
     this._div.style.position = "absolute";
     this._div.style.visibility = "hidden";
     this._div.style.zIndex = "-1";
-    document.body.appendChild(this._div);
+
+    if (this.props.isEmbeddingSdk) {
+      document
+        .getElementById(EMBEDDING_SDK_ROOT_ELEMENT_ID)
+        ?.appendChild(this._div);
```

Comment:
Should we console.warn/error if we don't find it?
I think its unlikely that this will break but it could but the could be hard to debug